source/MaterialXView/Viewer.cpp

```diff
@@ -741,6 +741,32 @@ void Viewer::saveActiveMaterialSource()
     }
 }
 
+void Viewer::loadActiveMaterialSource()
+{
+    try
+    {
+        MaterialPtr material = getSelectedMaterial();
+        mx::TypedElementPtr elem = material ? material->getElement() : nullptr;
+        if (elem)
+        {
+            std::string baseName = elem->getName();
+            std::string vertexShaderFile = _searchPath[0] / (baseName + "_vs.glsl");
```

Comment:
We might want to allow user input for where to save and load (like we do in Blacksky), as this could end up in places unexpected as searchPath[0] could be set to anything (it's a command line arg). For another change.

Reply:
Yes, for now this is just a debug feature for ourselves. searchPath[0] is what is being used when saving out using S-key, so when loading it's always the same file that was saved last for the current material.

But I agree we could improve on this later an make it more user facing.